### PR TITLE
Update pg-commander.rb

### PR DESCRIPTION
**Pull request by caarlos0**
Updated to version 1.5.7

### CODE REVIEW

File: Casks/pg-commander.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cask 'pg-commander' do
-  version '1.5.6'
-  sha256 '6793dc4b3c6b5fd77b615ae9eb29783d17d264b35f765793f938cef4eb60a852'
+  version '1.5.7'
+  sha256 'd5e48dea90d3060dd1f11f86101f953f6692e91340974fa7d88b85038e201240'
 
   # amazonaws.com is the official download host per the vendor homepage
   url "https://s3-eu-west-1.amazonaws.com/eggerapps.at/pgcommander/download/pgcommander-#{version}.zip"
```
Comment (vitorgalvao) [inline Casks/pg-commander.rb]: Looks like this needs changing. Travis is failing to get it.

**Comment by caarlos0** [comment]
@vitorgalvao done.

**Comment by vitorgalvao** [comment]
Thank you. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)? @mention me after you do, so I get an email to check back.

**Comment by caarlos0** [comment]
@vitorgalvao done

**Comment by vitorgalvao** [comment]
Thank you.

**Comment by caarlos0** [comment]
Thanks!

**Pull request merged**